src/data/sym/basic.lean

```diff
@@ -150,4 +159,111 @@ instance inhabited_sym' [inhabited α] (n : ℕ) : inhabited (sym' α n) :=
 
 end inhabited
 
+instance has_zero {α : Type*} : has_zero (sym α 0) := ⟨⟨0, rfl⟩⟩
+instance has_emptyc {α : Type*} : has_emptyc (sym α 0) := ⟨0⟩
+
+instance subsingleton {α : Type*} {n : ℕ} [g : subsingleton α] : subsingleton (sym α n) :=
+⟨begin
+  unfreezingI { cases g },
```

Comment:
Instead of using `cases` here, it would be more idiomatic to use `subsingleton.elim` instead of `g` below.